int32 BICYCLE = 0
int32 CAR = 1
int32 BUS =2 
